classes/fullstack/project1/client/src/App.tsx

```diff
@@ -114,14 +115,15 @@ function App() {
     return <div>Error fetching builds: {errorBuildFetch}</div>;
   }
 
-  if (builds.length === 0) {
-    return <div>Loading...</div>;
+  if (!Array.isArray(builds) || builds.length === 0) {
```

Comment:
use builds?.length or build?.[0].builId no point in using 2 boolean expressions here

Reply:
It was getting me errors just to run the app, this Boolean expression made it work somehow